ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-962] Prevent new dependant blank row when unnecessary

## Description of change
Does not show a new empty dependants field when returning to the dependants form

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAPP-962

## Notes for reviewer
N/A

## How to manually test the feature
Go through client income (earn more than 12K, has some savings, etc)

Enter dependants

Save and come back later

Do income journey again, the dependants form should be as last completed, without a new extra field
